superset/models/core.py

```diff
@@ -99,6 +100,11 @@ class CssTemplate(Model, AuditMixinNullable):
     css = Column(Text, default="")
 
 
+class ConfigurationMethod(str, enum.Enum):
+    SQLALCHEMY_URI = "sqlalchemy_form"
```

Comment:
can we make the var equal to the value
```suggestion
    SQLALCHEMY_FORM= "sqlalchemy_form"
```

Reply:
done, pushing up now.